### DISCUSSION TITLE
How to configure PreferThisOrSelfMethodCallRector to convert to static::

### DISCUSSION THREAD

**Discussion by alexander-schranz** [rectorphp/rector#7326] [Q&A]
I could not find how to configure the `PreferThisOrSelfMethodCallRector` rule to conver to static.

It tells: `Changes $this->... and static:: to self:: or vise versa for given type`. So I think I can also convert to `static::` or is that not possible?

**Comment by TomasVotruba** [comment]
This rule handles mainly PHPUnit test case calls pattern, see:

https://github.com/rectorphp/rector/blob/main/docs/rector_rules_overview.md#preferthisorselfmethodcallrector

Somebody prefer `$this->assert(...)` and somebody else prefer `self::assert(...)`. This rule helps to unite it across the code base :)
